src/views/tutorial-view/components/next-previous/components/directional-link-box/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import Link, { LinkProps } from 'next/link'
+import InlineSvg from '@hashicorp/react-inline-svg'
+import s from './directional-link-box.module.css'
+
+interface DirectionalLinkBoxProps {
+  link: LinkProps
+  label: string
+  title: string
+  direction: DirectionOption
+}
+
+type DirectionOption = 'next' | 'previous' | 'final'
+
+const IconSrcDict: { [k in DirectionOption]: string } = {
+  next: require(`@hashicorp/flight-icons/svg/arrow-right-16.svg?include`),
+  previous: require(`@hashicorp/flight-icons/svg/arrow-left-16.svg?include`),
+  final: require(`@hashicorp/flight-icons/svg/sliders-24.svg?include`),
+}
+
+/*
+ * @TODO style to spec
+ * look at the `CardLink` component to use for base styles!
+ */
+
+function DirectionalLinkBox({
+  link,
+  label,
+  title,
+  direction,
+}: DirectionalLinkBoxProps) {
+  return (
+    <Link href={link.href} as={link.as}>
+      <a className={s.linkbox} data-direction={direction}>
+        <span className={s.icon} data-direction={direction}>
+          <InlineSvg src={IconSrcDict[direction]} />
+        </span>
+        <span className={s.text}>
+          <span aria-hidden="true">{label}</span>
+          <span className={s.hidden}>{title}</span>
```

Comment:
Am I right that `label` is the visual label and `title` is the screen reader label?

Reply:
Yes! That is right

Reply:
I think the approach at adding an accessible label on these components needs to be different, but it can be addressed the same PR as when `CardLink` is brought in. I think it'll be easier there. The gist is that the easiest and most backwards-compatible way to set an accessible label is here is by setting `aria-label` on the `<a>` element.